src/libYARP_math/src/math.cpp

```diff
@@ -788,6 +788,7 @@ Vector yarp::math::dcm2rpy(const Matrix &R)
         else
         {
             // Not a unique solution: psi-phi=atan2(-R12,R11)
+            singularity = true;
```

Comment:
No idea what this does... @traversaro can you review this part?

Reply:
copied the structure from https://github.com/robotology/yarp/blob/f158a2436c8897a81ba599071f3c5f9c366f8d7e/src/libYARP_math/src/math.cpp#L737

Reply:
It is ok!